src/pages/PageMain.vue

```diff
@@ -1,26 +1,33 @@
 <template>
-    <v-app>
-        <navigation />
+  <v-app>
+    <Navigation />
 
-        <v-toolbar color="primary" dark fixed app>
-            <v-toolbar-side-icon @click.stop="drawer = !drawer"></v-toolbar-side-icon>
-            <v-toolbar-title>Bicycle Library</v-toolbar-title>
-        </v-toolbar>
+    <v-toolbar color="primary" dark fixed app>
+      <v-toolbar-side-icon @click.stop="toggleDrawer"></v-toolbar-side-icon>
+      <v-toolbar-title>Bicycle Library</v-toolbar-title>
+    </v-toolbar>
 
-        <v-content fill-height>
-            <router-view></router-view>
-        </v-content>
-    </v-app>
+    <v-content fill-height>
+      <router-view></router-view>
+    </v-content>
+  </v-app>
 </template>
 
 <script>
 import Navigation from '../components/Navigation.vue'
```

Comment:
```suggestion
import Navigation from '@/components/Navigation.vue'
```
Since this already start from root, use @ for easier readability. This is more of a test for the suggestion feature than anything else.